src/UKHO.ERPFacade.API/Helpers/LicenceUpdatedSapMessageBuilder.cs

```diff
@@ -72,12 +71,12 @@ private SapRecordOfSalePayLoad SapXmlPayloadCreation(RecordOfSaleEventPayLoad ev
                 Users = eventData.Data.Licence.NumberLicenceUsers,
                 EndUserId = eventData.Data.Licence.LicenceId,
                 ECDISMANUF = eventData.Data.Licence.Upn,
-                OrderNumber = eventData.Data.Licence.TransactionType == TransactionType ? "" : eventData.Data.Licence.OrderNumber,
-                StartDate = eventData.Data.Licence.TransactionType == TransactionType ? "" : eventData.Data.Licence.OrderDate,
-                PurachaseOrder = eventData.Data.Licence.TransactionType == TransactionType ? "" : eventData.Data.Licence.PoRef,
-                EndDate = eventData.Data.Licence.TransactionType == TransactionType ? "" : eventData.Data.Licence.HoldingsExpiryDate,
-                LicenceType = eventData.Data.Licence.TransactionType == TransactionType ? "" : eventData.Data.Licence.LicenceType,
-                LicenceDuration = eventData.Data.Licence.TransactionType == TransactionType ? null : eventData.Data.Licence.LicenceDuration
+                OrderNumber = eventData.Data.Licence.TransactionType == ChangeLicenceType ? "" : eventData.Data.Licence.OrderNumber,
+                StartDate = eventData.Data.Licence.TransactionType == ChangeLicenceType ? "" : eventData.Data.Licence.OrderDate,
+                PurachaseOrder = eventData.Data.Licence.TransactionType == ChangeLicenceType ? "" : eventData.Data.Licence.PoRef,
+                EndDate = eventData.Data.Licence.TransactionType == ChangeLicenceType ? "" : eventData.Data.Licence.HoldingsExpiryDate,
```

Comment:
Please remove check of TransactionType and assign empty values as its particular to License change event